src/components/Avatar/Avatar.stories.tsx

```diff
@@ -5,7 +5,7 @@ export default { component: Avatar, title: 'Avatar' };
 
 export const Sizes = () => {
 	return (
-		<>
+		<React.Fragment>
```

Comment:
As above, can we use[`@babel/plugin-transform-react-jsx
`](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx#usage) to avoid this?

Reply:
This is part of the [React Babel Preset](https://babeljs.io/docs/en/babel-preset-react), I would expect it to be already available.

Reply:
I suspect that the problem is with the elderly version of `@babel/preset-react` that we're currently using. @liywjl can you try upgrading to v7.9.0 or later?